coredns/README.md

```diff
@@ -0,0 +1,68 @@
+# CoreDNS Integration
+
+## Overview
+Get metrics from CoreDNS in real time to:
+
+* Visualize and monitor DNS failures and cache hits/misses
+## Setup
+
+### Installation
+1. Download the Datadog Agent.
+2. Download the `datadog_checks/coredns/coredns.py` file.
+3. Place it in the Agent's checks.d directory.
+
+### Configuration
+1. Download the `datadog_checks/coredns/data/conf.yaml.example`
+2. Place file in the Datadog Agent's `conf.d` directory.
+3. Rename file to coredns.yaml
+4. Edit file to point to your server and port.
+5. [Restart the Agent][3] to begin sending CoreDNS metrics to Datadog.
+
+### Validation
+
+[Run the Agent's `status` subcommand][4] and look for `coredns` under the Checks section:
+
+```
+  Checks
+  ======
+    [...]
+
+    coredns
+    -------
+      - instance #0 [OK]
+      - Collected 26 metrics, 0 events & 1 service check
+
+    [...]
+```
+
+## Compatibility
```

Comment:
No need to add the compatibility section in the readme. The information is already in the manifest.json file and has a specific display on the public documentation:

https://docs.datadoghq.com/integrations/apache/